recipes/pggb/meta.yaml

```diff
@@ -10,26 +10,26 @@ source:
   sha256: f443a6354f30307573545d03c7491de299ca50dfcba2a12832fb77e0452e46f4
 
 build:
-  number: 0
+  number: 1
   skip: true # [osx]
   run_exports:
   - {{ pin_subpackage(name, max_pin='x.x') }}
 
 requirements:
   run:
-    - wfmash ==0.14.0
+    - wfmash
```

Comment:
We should not change those pins without asking the maintainer ... ping @AndreaGuarracino